content/docs/mutating-webhook/external-vault.md

```diff
@@ -37,7 +37,7 @@ Basically, you have to grant `cluster2` access to the Vault running on `cluster1
     kubectl get secret $(kubectl get sa vault -o jsonpath='{.secrets[0].name}') -o jsonpath='{.data.token}' | base64 --decode
     ```
 
-1. In the `vault.banzaicloud.com` custom resource (for example, [https://github.com/bank-vaults/bank-vaults/raw/main/operator/deploy/cr.yaml](https://github.com/bank-vaults/bank-vaults/raw/main/operator/deploy/cr.yaml)) of `cluster1`, define an `externalConfig` section. Fill the values of the `kubernetes_ca_cert`, `kubernetes_host`, and `token_reviewer_jwt` using the data collected in the previous steps.
+1. In the `vault.banzaicloud.com` custom resource (for example, [https://github.com/bank-vaults/vault-operator/blob/main/deploy/examples/cr.yaml](https://github.com/bank-vaults/vault-operator/blob/main/deploy/examples/cr.yaml)) of `cluster1`, define an `externalConfig` section. Fill the values of the `kubernetes_ca_cert`, `kubernetes_host`, and `token_reviewer_jwt` using the data collected in the previous steps.
```

Comment:
You're right, but actually it would be even better if we could add an actual example cr yaml that has these values configured. Do you have such an example at hand?

Reply:
The content of the linked file can be copied here, but then if that changes, we have to keep in mind to update it here as well...